src/include/catalog/manager.h

```diff
@@ -21,13 +21,13 @@
 
 #include "common/macros.h"
 #include "common/types.h"
-#include "container/cuckoo_map.h"
 #include "container/lock_free_array.h"
 
 namespace peloton {
 
 namespace storage {
 class TileGroup;
+class IndirectionArray;
 }
 
 namespace catalog {
```

Comment:
the manager is responsible for the allocation of tile, tile group, and indirection layer. So we do not need to rename it.